README.md

```diff
@@ -1069,9 +1069,15 @@ yet but might provide some value.
   end
   ```
 
-### Style Guides
+### Alternative Style Guides
 
-Check [Awesome Elixir][Style Guides] for a list of alternative style guides.
+* [Aleksei Magusev's Elixir Style Guide]
+  — An opinionated Elixir style guide. Developed by some of the Elixir's core
+  members and used in the Elixir project itself.
```

Comment:
"stemed" -> "stemmed" and "Elixir's core team" -> "the Elixir core team", then it's perfect for me :)